nilearn/reporting/get_clusters_table.py

```diff
@@ -433,3 +435,78 @@ def get_clusters_table(
         result_table = pd.DataFrame(columns=cols, data=rows)
 
     return (result_table, label_maps) if return_label_maps else result_table
+
+
+def clustering_params_to_dataframe(
```

Comment:
Previously, this was under `_utils`, so should this be prefixed with a `_`?
Just asking to make sure...

Reply:
na it's used in several modules, including some outside of reporting, so no leading underscore